force-app/main/default/classes/LeadTriggerHandler.cls

```diff
@@ -32,17 +32,32 @@ public with sharing class LeadTriggerHandler {
 	 */
 	public static void handleTitleNormalization(List<Lead> leadsToNormalize) {
 		for (Lead ld : leadsToNormalize) {
-			if (ld.title == 'vp' || ld.title.contains('v.p.') || ld.title.contains('vice president')) {
+			if (String.isBlank(ld.Title)) {
+				continue;
+			}
+			if (
+				ld.title.containsIgnoreCase('vp') ||
+				ld.title.containsIgnoreCase('v.p.') ||
+				ld.title.containsIgnoreCase('vice president')
+			) {
 				ld.Title = 'Vice President';
 			} else if (
-				ld.title.contains('mgr') ||
-				ld.title.contains('manage') ||
-				ld.title.contains('head of department')
+				ld.title.containsIgnoreCase('mgr') ||
+				ld.title.containsIgnoreCase('manage') ||
+				ld.title.containsIgnoreCase('head of department')
 			) {
 				ld.Title = 'Manager';
-			} else if (ld.title.contains('exec') || ld.title == 'chief' || ld.title.contains('head')) {
+			} else if (
+				ld.title.containsIgnoreCase('exec') || 
+				ld.title.containsIgnoreCase('chief') || 
+				ld.title.containsIgnoreCase('head')
+			) {
 				ld.Title = 'Executive';
-			} else if (ld.title.contains('assist') || ld.title.contains('deputy') || ld.title == 'jr') {
+			} else if (
+				ld.title.containsIgnoreCase('assist') || 
+				ld.title.containsIgnoreCase('deputy') || 
+				ld.title.containsIgnoreCase('jr')
```

Comment:
For all of these examples try create a contains the values that you are looking for then do contain on the list.